systemtests/tests/python-bareos/test_list_command.py

```diff
@@ -239,6 +239,12 @@ def test_list_jobs(self):
         for job in result["jobs"]:
             self.assertTrue(job["jobstatus"], "T")
 
+        # list jobs jobstatus=X,Y,z
+        result = director.call("list jobs jobstatus=T,E,f")
+        self.assertTrue(result["jobs"])
+        for job in result["jobs"]:
+            self.assertTrue(job["jobstatus"] == "T" or job["jobstatus"] == "E" or job["jobstatus"] == "f")
+
```

Comment:
This is not going to test for any job statuses other than `T`.
The test does only produce terminated jobs, so there is no chance that you can find an `E` or `f`.
You'll have to provoke those statuses (or others) first before testing if the call can find them.

Reply:
isn't the director call supposed to trigger list jobs with multi variable jobstatus so that's supposed to send back jobs that have any jobstatus from the input ( the new functionality basically )